core/src/main/java/de/jplag/merging/MergingParameters.java

```diff
@@ -2,17 +2,17 @@
 
 /**
  * Collection of parameters that describe how a match merging should be performed.
- * @param mergeBuffer describes how shorter a match can be than the Minimum Token Match (Defaults to 0).
- * @param seperatingThreshold describes how many tokens can be between to neighboring matches (Defaults to 0).
+ * @param neighborLength describes how short a match can be, to be considered (Defaults to 2).
+ * @param gapSize describes how many tokens can be between to neighboring matches (Defaults to 6).
  */
-public record MergingParameters(boolean enabled, int mergeBuffer, int seperatingThreshold) {
+public record MergingParameters(boolean enabled, int neighborLength, int gapSize) {
```

Comment:
`minimumNeighborLength` would match the naming scheme of the `minimumTokenMatch`.

Reply:
What about the cli option?
Wont `--minimum-neighbor-length` be too long

Reply:
Following the same logic, `gapSize` should be `maximumGapSize`, which would get tedious to write.
Ultimately it is your decision.

Reply:
So `maximumGapSize`, `--maximum-gap-size` `minimumNeighborLength`and `--minimum-neighbor-length` ?

Reply:
for the CLI flags, we could leave minimum and maximum away, as we can add it directly to the flag description. They are more often written than read.